learning/tour-of-beam/frontend/lib/config/theme/switch_notifier.dart

```diff
@@ -34,21 +33,24 @@ class ThemeSwitchNotifier extends ChangeNotifier {
   ThemeColors get themeColors {
     if (themeMode == ThemeMode.dark) {
       return _darkThemeColors;
-    } else {
-      return _lightThemeColors;
     }
+    return _lightThemeColors;
   }
 
   void init() {
     _setPreferences();
   }
 
-  Future<void> _setPreferences() async {
-    _preferences = await SharedPreferences.getInstance();
-    themeMode = _preferences.getString(kThemeMode) == ThemeMode.dark.toString()
-        ? ThemeMode.dark
-        : ThemeMode.light;
-    notifyListeners();
+  void _setPreferences() {
+    SharedPreferences.getInstance().then(
```

Comment:
https://medium.com/flutter-senior/why-use-async-await-instead-of-future-then-2e9c340aabfb